components/afObjectField/afObjectField.js

```diff
@@ -3,5 +3,10 @@
 Template.afObjectField.helpers({
   getTemplateName: function () {
     return AutoForm.getTemplateName('afObjectField', this.template, this.name);
+  },
+  innerContext: function() {
+    var c = AutoForm.Utility.getComponentContext(this, 'afObjectField');
+    _.extend(this, c.atts);
+    return this;
```

Comment:
We should never mutate `this`. Maybe do `return _.extend({}, this, c.atts);` instead.